docs/get-started/backup/restore_MongoDB_backups.md

```diff
@@ -0,0 +1,126 @@
+# Restore a MongoDB backup
+
+## Restore compatibility
+
+MongoDB backups can only be restored to the same service they were created from.
+
+To restore a backup:
```

Comment:
Let's add a title above this like cuz it look like it's a part of `Restore compatibility` section.